ui/src/utils/testStatus.test.ts

```diff
@@ -0,0 +1,19 @@
+import { getTestStatus } from './testStatus';
+
+describe('getTestStatus', () => {
+  it(`should identity a running test`, () => {
+    const obtainedStatus = getTestStatus(true);
```

Comment:
What do you think about assigning the `true` here to a `const` called `isEnabled` or similar? I think that might make it's intention clearer.

Reply:
Good idea, thanks!